kolibri/utils/main.py

```diff
@@ -78,6 +78,10 @@ def conditional_backup(kolibri_version, version_file_contents):
 
         try:
             backup = dbbackup(version_file_contents)
+            backups = get_backup_files(kolibri_version)
```

Comment:
As I commented above, I think the logic here is inverted - we should be getting all the backup files not for this version, as that will be the older backups.